ipa-core/proptest-regressions/ff/prime_field.txt

```diff
@@ -0,0 +1,8 @@
+# Seeds for failure cases proptest has generated in the past. It is
+# automatically read and these particular cases re-run before any
+# novel cases are generated.
+#
+# It is recommended to check this file in to source control so that
+# everyone who runs the test benefits from these saved cases.
+cc 928efec8e7e6e629845d55708bfa6250016460758d23808b9022961a3ee05f7c # shrinks to mut elements = [0_mod31]
```

Comment:
we probably don't need to commit this - seed shows that it just generates an empty input which proptest always does

Reply:
ah sorry, I missed that that got generated